content/concepts/introduction/technical-faq.md

```diff
@@ -0,0 +1,99 @@
+# Technical FAQ
+
+## Validators and Unique Node Lists
+
+<!--#{ using h4s for questions to keep them out of the right side nav (too cluttered when they display) and to provide appropriate text size for questions. #}-->
+#### What service do transaction validators provide?
+
+Validators determine if transactions meet protocol requirements, and are therefore “valid.” The service validators uniquely provide is grouping transactions into ordered units, agreeing on one such ordering specifically to prevent double spending.
+
+See [The Ripple Ledger Consensus Process](x) and the <a href="https://ripple.com/insights/ripple-labs-tech-talk-consensus-within-the-ripple-protocol/" target="_blank">Ripple Labs Tech Talk: Understanding Consensus <i class="fa fa-external-link" aria-hidden="true"></i></a><!--#{ fix md highlighting_ #}--> for more information about the consensus process.
+
+
+#### How much does it cost to run a validator?
+
+Running a validator does not require any fees or XRP. It is comparable in cost to running an email server in terms of electricity.
+
+
+#### What are Unique Node Lists (UNLs)?
+
+They are the lists of transaction validators a given participant believes will not conspire to defraud them.
+
+
+#### Which UNL should I select?
+
+Since anybody can run a validator, the burden is on the network participants to choose a reliable set. Currently, _Ripple_ (further mentions of Ripple in this document that are italicized to represent the company) provides a default and recommended list which we expand based on watching the history of validators operated by _Ripple_ and third parties. Eventually, _Ripple_ intends to remove itself from this process entirely by having network participants select their own lists based on publicly available data about validator quality.
+<!--#{ TODO: See the references to _Ripple_ the company vs Ripple (non-ital) as what I think is the XRP Ledger. Should we just replace all references to the non-italicized Ripple with "XRP Ledger" or "`rippled`" as appropriate? #}-->
```

Comment:
> Should we just replace all references to the non-italicized Ripple with "XRP Ledger" or "`rippled`" as appropriate?

Yes.

After doing so, we can also de-italicize _Ripple_ and remove the note explaining that it refers to the company.